docs/logaxis.md

```diff
@@ -0,0 +1,27 @@
+## jquery.flot.logaxis
+This plugin is used to create logarithmic axis. This includes tick generation,
+formatters and transformers to and from logarithmic representation.
```

Comment:
Are those transform functions public? If yes then I'm not seeing them documented.